### PR TITLE
ZK-4827: Fail to load JSP with ZK Components

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zk/src/archive/web/js/zk/mount.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ function zkpi(nm, wv) {
 
 //ZK JSP: page creation (backward compatible)
 function zkpb(pguid, dtid, contextURI, updateURI, resourceURI, reqURI, props) {
+	if (reqURI === undefined || typeof reqURI !== 'string') { //ZK-4827: for backward compatible (other addons, ex. zuljsp)
```
Comment (DevChu) [inline zk/src/archive/web/js/zk/mount.js:24]: if (props === undefined && typeof reqURI !== 'string')

**Review by DevChu — COMMENTED** [review]
test get zkau/image.jpg when setting zkres

**Review by scribetw — CHANGES_REQUESTED** [review]
fix zkpb and publish new zuljsp

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**